sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -225,7 +224,6 @@ def state(self) -> str:
         """
         return self._state
 
-    @experimental
```

Comment:
We should add this change under changelog.

Reply:
Added.